### PR TITLE
Mobile 3810

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

File: tests/behat/behat_app.php [hunk 1]
```diff
@@ -553,7 +536,7 @@ function($context, $args) {
     /**
      * Presses standard buttons in the app.
      *
-     * @Given /^I press the (back|main menu|page menu|accounts menu) button in the app$/
+     * @Given /^I press the (back|more menu|page menu|user menu) button in the app$/
```
Comment (NoelDeMartin) [inline tests/behat/behat_app.php:539]: If we want `main menu` to continue working, we should leave it here as one of the possibilities.

File: tests/behat/app_behat_runtime.js [hunk 2]
```diff
@@ -545,28 +473,22 @@
...
-                    tooMany = true;
-                }
-            }
+        let backdrops = Array.from(document.querySelectorAll('ion-backdrop'));
+        backdrops = backdrops.filter(function(backdrop) {
+            return !!backdrop.offsetParent;
         });
-        if (!found) {
+
+        if (backdrops.length < 0) {
```
Comment (NoelDeMartin) [inline tests/behat/app_behat_runtime.js:484]: Length will never be lower than 0, you could do `!backdrops.length` like you've done with the titles below or just `backdrops.length === 0`

**Review by NoelDeMartin — CHANGES_REQUESTED** [review]
I found a couple of things to fix, I think it's ready to merge once those are amended.

File: tests/behat/app_behat_runtime.js (hunk 2 above)
Comment (crazyserver) [reply tests/behat/app_behat_runtime.js]: Solved thanks!

File: tests/behat/behat_app.php (hunk 1 above)
Comment (crazyserver) [reply tests/behat/behat_app.php]: Solved thanks!

**Review by NoelDeMartin — APPROVED** [review]

**Pull request merged**